cmd/minikube/cmd/root.go

```diff
@@ -115,8 +118,11 @@ func setFlagsUsingViper() {
 
 func init() {
 	RootCmd.PersistentFlags().Bool(showLibmachineLogs, false, "Show logs from libmachine.")
-	RootCmd.PersistentFlags().String(username, "", "User name for the virtual machine.")
-	RootCmd.PersistentFlags().String(password, "", "Password for the virtual machine.")
+	RootCmd.PersistentFlags().String(username, "", "User name for the virtual machine registration.")
+	RootCmd.PersistentFlags().String(password, "", "Password for the virtual machine registration.")
+	RootCmd.PersistentFlags().String(httpProxy, "", "HTTP proxy for virtual machine.")
+	RootCmd.PersistentFlags().String(httpsProxy, "", "HTTPS proxy for virtual machine.")
```

Comment:
Do we really need two proxy options? Given the format we require, the distinction between HTTP and HTTPS is implicit right? Different if you just allow the user to do something like HTTPS_PROXY=<hostname>, then the protocol cannot be derived. WDYT?